File: android/lib/src/main/java/duck/cameras/android/model/Camera.java
package duck.cameras.android.model;

import java.io.Serializable;
import java.util.ArrayList;

public class Camera implements Serializable {
    public String endPoint;
    public ArrayList<Service> services;
    public Information information;
    public ArrayList<Profile> profiles;

    public static class Service implements Serializable {
        public String namespace;
        public String address;
        public String version;
    }

    public static class Information implements Serializable {
        public String manufacturer;
        public String model;
        public String firmwareVersion;
        public String serialNumber;
        public String hardwareId;
    }

    public static class Profile implements Serializable {
        public String name;
        public String token;
        public String streamUri;
        public String snapshotUri;
    }

    public Camera() {
        this.services = new ArrayList<>();
        this.information = new Information();
        this.profiles = new ArrayList<>();
    }
}

File: windows/Model/Camera.cs
using System.Collections.Generic;

namespace Duck.Cameras.Windows.Model
{
    public class Camera
    {
        public string EndPoint;
        public List<CameraService> Services;
        public CameraInformation Information;
        public List<CameraProfile> Profiles;
        public string Name;

        public Camera()
        {
            Services = new List<CameraService>();
            Information = new CameraInformation();
            Profiles = new List<CameraProfile>();
        }
    }

    public class CameraService
    {
        public string Namespace;
        public string Address;
        public string Version;
    }

    public class CameraInformation
    {
        public string Manufacturer;
        public string Model;
        public string FirmwareVersion;
        public string SerialNumber;
        public string HardwareId;
    }

    public class CameraProfile
    {
        public string Name;
        public string Token;
        public string StreamUri;
        public string SnapshotUri;
    }
}